releasenotes/notes/include-ecr-gates-for-pulse-scaling-8369eb584c6d8fe1.yaml

```diff
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    This PR now allows the pulse scaling techniques found in RZXCalibrationBuilder
+    and RZXCalibrationBuilderNoEcho to consume `ecr` entangling gates from the
+    backend, in addition to the `cx` gates they were build for. These native gates
+    contain the calibrated pulse schedules that the pulse scaling passes use to 
+    generate arbitraty rotations of the :class:`~RZXGate` operation.
+issues:
+  - |
+    Currently the FakeSherbrooke backend does not contain `.defaults()` from which
+    the instruction schedule map is obtained. The unit test `TestCalibrationBuilder` 
+    requires this to check `ecr` operations work similarly to the `cx` gate case. 
+    As of this PR, `ibm_sherbrooke` is the only backend with native `ecr` gates, so 
+    this requires updating. 
```

Comment:
I think this should be resolved. The `FakeSherbrooke` backend is based on `BackendV2` which doesn't have `configuration()`, `properties()`, or `defaults()` methods. The calibrations for gates are part of the `Target` object (`backend.target`) and you can get an `InstructionScheduleMap` from a target (`backend.target.instruction_schedule_map()`). This pass was updated in #9343 to take in a target directly so when you run with `FakeSherbrooke` you should just be able to do:
```python
from qiskit.transpiler.passes import RZXCalibrationBuilder
from qiskit.providers.fake_provider import FakeSherbrooke

cal_pass = RZXCalibrationBuilder(target=FakeSherbrooke().target)
```

Reply:
This is not that easy. We need to build reference schedule to test, but currently the builder is tied to the V1 accessors and we cannot generate the schedule with FakeV2. I wrote [this issue](https://github.com/Qiskit/qiskit-terra/issues/9420) and assigned a team member to it. We plan to update the builder in 0.24, and then we can write a test with FakeSherbrooke. The plan is (since @nbronn needs this code in a few weeks) to wait for 0.23 release and merge this PR without test. Then add the test with followup PR after above issue is resolved.

Reply:
@nbronn You can remove this issue note. I don't think end-users have concern about unit-test. You can leave this as a comment in the unit-test file so that developer can be aware of this.